default.nix

```diff
@@ -81,9 +81,9 @@ in let
     };
 
     # Change the compiler when updating our own resolver.
-    compiler-nix-name = "ghc8107";
+    compiler-nix-name = "ghc925";
     ghcVersion = "8.10.7";
```

Comment:
This seems wrong... Or maybe this line does not do anything, actually???

Reply:
I don't know the Nix stuff, so I'm reluctant to do anything to it that might break something. @goodlyrottenapple , `git grep 8\\.10\\.7` references to ghc 8.10.7 remaining in `default.nix`, `nix/fourmolu.nix.d/default.nix`, and `nix/hlint.nix.d/default.nix`. What if anything should be done with them?

Should the reference in `README.md` (for building on ARM64 Darwin) simply be changed to say `ghc-9.2.5`, or is there something more that needs to be done there? I don't have access to such a system to experiment.

Reply:
@goodlyrottenapple `flake.nix` additionally includes `compiler-nix-name ? "ghc8107", profiling ? false`.

Reply:
Right, but how? I don't know which sections need to be changed and which need to be deleted.

Reply:
Maybe we just change all of them for now, that seems safe to do. We simply don't want any ghc-8 in our setup any more.